tuf/src/interchange/cjson/shims.rs

```diff
@@ -2,6 +2,7 @@ use chrono::offset::Utc;
 use chrono::prelude::*;
 use serde_derive::{Deserialize, Serialize};
 use std::collections::{BTreeMap, HashSet};
+use std::convert::{TryFrom, TryInto};
```

Comment:
is `TryInto` needed here? it's included in edition 2021 prelude?

Reply:
done